app_flutter/lib/main.dart

```diff
@@ -8,14 +8,17 @@ import 'build_dashboard.dart';
 
 void main() => runApp(MyApp());
 
+ThemeData theme = ThemeData(
```

Comment:
```suggestion
final ThemeData theme = ThemeData(
```

Reply:
Thanks! It's weird that the linter didn't complain about this.